server/datastore/mysql/software_titles.go

```diff
@@ -299,16 +318,17 @@ GROUP BY st.id, package_self_service, package_name, package_version, vpp_app_sel
 		`, cveJoinType)
 	}
 
+	var args []any
 	if match != "" {
 		additionalWhere = " (st.name LIKE ? OR scve.cve LIKE ?)"
 		match = likePattern(match)
 		args = append(args, match, match)
 	}
 
 	// default to "a software installer or VPP app exists", and see next condition.
-	defaultFilter := `
-		(si.id IS NOT NULL OR vat.adam_id IS NOT NULL)
-	`
+	defaultFilter := fmt.Sprintf(`
+		((si.id IS NOT NULL OR vat.adam_id IS NOT NULL) AND %s)
+	`, notExcludeVPPAppsAndSoftwareInstallers)
```

Comment:
This is for "All teams" software view to not show installers/VPP-apps that are not installed, see
https://github.com/fleetdm/fleet/issues/19447#issuecomment-2270006803

Reply:
This one is a little tough to follow on first reading with the double negatives. Could we name the variable `includeVPPAppsAnd...` and initialize it to "TRUE"?

Reply:
+1! done.